### PR TITLE
a fluix fix

### PR DESCRIPTION
**Pull request by Agrax**

### CODE REVIEW

File: scripts/applied_energistics_2.zs [hunk 1]
```diff
@@ -1178,14 +1180,29 @@ alloy_smelter.recipeBuilder()
...
+chemical_reactor.recipeBuilder().inputs(<ore:dustFluix> * 1, reddosanudo * 1).fluidInputs(<liquid:water> * 100).outputs(seedCrystal * 2).duration(100).EUt(30).buildAndRegister();
+
+// --- Autoclave recipes ---
+
+// Pure fluix crystals
+autoclave.recipeBuilder().inputs(seedCrystal * 1).fluidInputs(<liquid:water> * 1000).chancedOutput(PureFluixCrystal * 1, 6000, 500).duration(2000).EUt(24).buildAndRegister();
+autoclave.recipeBuilder().inputs(seedCrystal * 1).fluidInputs(<liquid:distilled_water> * 1000).chancedOutput(PureFluixCrystal * 1, 7000, 800).duration(1000).EUt(24).buildAndRegister();
+// Fluix crystals
+autoclave.recipeBuilder().inputs(<ore:dustFluix> * 1).fluidInputs(<liquid:water> * 1000).chancedOutput(FluixCrystal * 1, 7000, 600).duration(2000).EUt(24).buildAndRegister();
+autoclave.recipeBuilder().inputs(<ore:dustFluix> * 1).fluidInputs(<liquid:distilled_water> * 1000).chancedOutput(FluixCrystal * 1, 7500, 900).duration(1000).EUt(24).buildAndRegister();
```
Comment (pyure) [inline scripts/applied_energistics_2.zs]: This tier bonus means after only 3 tiers it will output 100%.  Do you care?

**Comment by Agrax** [comment]
In IT3 there’s only tiers LV through IV. So, 5 tiers. Lets’s take 70% for example:
70, 78, 86, 94, 100

Is it *that* bad?

**Comment by TedKnudsen** [comment]
Thanks, revision looks good.

**Comment by pyure** [comment]
Thanks revision looks good.

**Pull request merged**